src/tslint-rules.json

```diff
@@ -109,10 +109,10 @@
 			"parameter",
 			"arrow-parameter",
 			"property-declaration",
-			"variable-declaration",
-			"member-variable-declaration",
-			"object-destructuring",
-			"array-destructuring"
+//			"variable-declaration",
```

Comment:
Why comment rather than delete the rules outright?
The values are well documented and can easily be added later on, if necessary.

Reply:
Well, other lines are commented so I thought that's the way to be. Also it's easier to revert if needed be.